src/index.js

```diff
@@ -87,11 +87,20 @@ class ExtractTextPlugin {
     }
   }
 
-  static renderExtractedChunk(chunk) {
+  static renderExtractedChunk(compilation, chunk) {
     const source = new ConcatSource();
 
     for (const chunkModule of chunk.modulesIterable) {
-      let moduleSource = chunkModule.source();
+      let moduleSource = chunkModule.source(
```

Comment:
https://github.com/webpack/webpack/pull/6597

Reply:
That workaround breaks for other types of Module; it's best to just treat these 2 arguments as part of the Module API now.